site/docs/v1/tech/apis/tenants.adoc

```diff
@@ -74,6 +75,64 @@ include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_tenant-response-body.adoc[]
 
+
+== Create a New Tenant From an Existing Tenant
+
+This API is used to create a new Tenant from an existing Tenant.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a Tenant with a randomly generated Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/tenant#
+--
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a Tenant with the provided unique Id
```

Comment:
```suggestion
link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Tenant from an existing Tenant with the given Id
```

Reply:
We use `provided unique` interchangeably with `given` in our docs.  Is there a preference here?

Reply:
I was only comparing the changes for this PR. I doubt whether anyone would notice that these are phrased differently if they weren't seeing them all side by side.